.github/workflows/validation.yml

```diff
@@ -8,6 +8,12 @@ on:
     branches:
       - "master"
 
+concurrency:
+  # If there's a PR number, it will result in cancelling the previous build for that PR
+  # this is needed because the PR event triggers on "meta" PR things, like comments but not pushes
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
```

Comment:
There’s probably also a “number of concurrent jobs” setting that defaults to 1, right? If so, please make it explicit.

Reply:
You'd think so, but there's no setting like that
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency